changelog/20.0/20.0.0/summary.md

```diff
@@ -10,6 +10,7 @@
     - [Delete with Subquery Support](#delete-subquery)
   - **[Flag changes](#flag-changes)**
     - [`pprof-http` default change](#pprof-http-default)
+    - [`shutdown_grace_period` Default Change](#shutdown-grace-period-default)
```

Comment:
We should create a section called Breaking Changes and move this there.

Reply:
Done